var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -440,6 +441,14 @@ def libs(self):
         when="@1.82.0 platform=windows",
     )
 
+    # https://github.com/boostorg/context/pull/280
```

Comment:
Where did you find this one? I don't see it on the [history](https://www.boost.org/users/history/version_1_87_0.html) page.

Reply:
Boost simply fails to build without it in spack (same error after bootstrap, on `--clean` as in the PR). I had reached about the same conclusion as the PR around the time when it appears.